src/components/institution/SelectInstitutionSkeleton.tsx

```diff
@@ -0,0 +1,22 @@
+import { Box, Skeleton, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+
+export const SelectInstitutionSkeleton = () => {
+  const { t } = useTranslation();
+  return (
+    <Box sx={{ width: '100%' }}>
+      <Box sx={{ width: '100%' }}>
+        <Typography variant="h3" sx={{ fontWeight: 'normal' }}>
+          {t('common.select_institution')}
```

Comment:
Er det uheldig at tekstinnholdet her også ligger i Skeleton? Antar samme tekst også ligger en annen plass for visning når institutsjon er hentet, så vi må huske å passe på at vi alltid har samme key begge plasser da

Reply:
samme med layout ellers også forsåvidt. Om det skal legges inn mer hjelpetekst e.l. her en gang, må det legges til både her og i `SelectInstitutionForm` ogs

Reply:
Ja, grunnen til at det er der er fordi hvis ikke vil alt bli flyttet på, inkludert størrelsen på boksen. Jeg kunne jo laget en komponent for teksten og importert begge plassene men det føles litt som overkill? Teksten er jo allerede kun på ett sted, så endrer vi den i tekstfila blir den endret begge plasser.